aries-site/src/layouts/main/SidebarLayout.js

```diff
@@ -27,8 +28,8 @@ export const SidebarLayout = ({ mainContentChildren }) => {
 
   // TODO selectedNav should be retrived from aries-core
   // as the selected element of the NavBar
-  const selectedNav = 'foundation';
-
+  const topic = 'Foundation';
```

Comment:
We are going to have to rework how this topic variable gets populated. It should get passed in as a prop from the component consuming `<SidebarLayout>` --- outside the scope of this PR though.

Reply:
Agreed.

Reply:
Agreed.